frontend/website/src/Links.re

```diff
@@ -46,6 +46,11 @@ module Static = {
     link: Cdn.url("/static/coda-whitepaper-05-10-2018-0.pdf"),
   };
 
+  let modifiedsnark = () => {
+    name: ("Read the zkSNARK We Are Using", "modified-BG-snark"),
```

Comment:
Grammar: should be "Read about the zkSNARKs we're using". Or maybe the "Read about the zkSNARK construction we're using".

Reply:
Done.